pre_commit/clientlib.py

```diff
@@ -144,6 +147,16 @@ def _entry(modname):
     )
 
 
+def warn_on_unknown_keys_at_top_level(extra, orig_keys):
+    logger.warning(
+        'Your pre-commit-config contain these extra keys: {}. '
+        'while the only valid keys are: {}.'.format(
```

Comment:
let's keep this short and sweet, how about
```python
'Unexpected key(s): {}'.format(', '.join(sorted(extra)))
```

Reply:
are you sure it will be clear to the user what is the problem here?

Reply:
hmm then maybe `Unexpected config key(s): ...`?  The other message has context of location but it looks like we didn't think about that when adding `WarnAdditionalKeys` 🤔 https://github.com/pre-commit/pre-commit/blob/1c7ac67fba963b7a8cda852ed04bceb8a1c37063/pre_commit/repository.py#L108